### ISSUE TITLE
Investigate why the performance of models pre-trained with images is so bad.

### ISSUE THREAD

**Issue by t46** [t46/paper-pre-training-different-modality-offline-rl#4]
## Summary
In the `neurips-2022` version, the author investigated why the models pre-trained with image data fail to be fine-tuned. The conclusion was that large gradient norm and gradient clipping may badly affect the performance. However, these alone do not explain why the performance image-pre-trained model is so bad. Further exploration is necessary to identify the essential cause of the catastrophic performance.

## Reference
> The paper considers the number of aspects (e.g., activations, parameters, etc.) and reports interesting observations. However, it feels that it often stops too early and presents a hypothesis without trying to explore it further. For example, in the case of the gradient analysis the paper suggests that iGPT does not train well due to large gradients in early layers. It would be nice to explore this further and see if, e.g., this observation can inform a better training recipe for offline RL with iGPT pre-training.

https://openreview.net/forum?id=9GXoMs__ckJ&noteId=j43DHndu18D

> In section 5.4, the paper hypothesizes that the fact that the gradient confusion is higher for the pretrained models, that makes them hard to train which explains the smaller changes in parameters. But this does not again seem to affect the performance in a detrimental way.

https://openreview.net/forum?id=9GXoMs__ckJ&noteId=PWjgJYjylPh
